knimin/tests/test_ag_pulldown.py

```diff
@@ -53,6 +55,13 @@ def test_get(self):
             self.assertIn("<option value='%s'>%s</option>" % (survey, survey),
                           response.body)
         self.assertNotIn('<input type="submit" disabled>', response.body)
+        for (id, name, selected) in db.list_ag_surveys():
```

Comment:
ok, in that case what it means is that you don't need the if statement because all are selected. Given the get issued, all should be selected so the test should actually be:
```python
for (_id, name, selected) in db.list_ag_surveys():
    self.assertIn("<option value='%i' selected>%s</option>" % (_id, name), response.body)
```

Note also the change from `id` -> `_id`.